### PR TITLE
Feature/wire set dipswitch

### PR DESCRIPTION
**Pull request by neophob**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,12 @@ npm run test
 
 For more infos about how tests are written, see [here](https://github.com/vpdb/vpx-js/tree/master/test#readme)
 
+## VPX Fileformat
+
+VPX files using Microsoft OLE2 files (also called Structured Storage, Compound File Binary Format or Compound Document File Format). Some links to work (extract/repack) those files:
+- [oletools]: https://www.decalage.info/python/oletools
+- [Visual Pinball]: https://vpinball.com/
```
Comment (freezy) [inline README.md]: I don't think this adds any value to the main readme. And VP's official URL is https://sourceforge.net/projects/vpinball/

File: bin/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Command Line Tools
 
-VPX-JS comes with two command line tools:
+VPX-JS comes with command line tools. Make sure to run `npm run compile` first, then change to the compiled output directory (`dist/cjs/bin`) when you use Node.js. Another option is transpile the TypeScript file on the fly and execute it using `npx ts-node bin/vptscript.ts`.
```
Comment (freezy) [inline bin/README.md]: I don't think you need to do that if you get vpx-js from npm?

Comment (neophob) [reply bin/README.md]: i guess if you take the package from npm you don't read the readme. 

if we have some tools bundled with vpx-js, then it should be clear how run them - else it is not very helpful.

File: README.md (hunk 1 above)
Comment (neophob) [reply README.md]: VPX files is the main file for vpx-js, so not sure if I agree. But I can remove this if you don't agree

**Comment by freezy** [comment]
Thanks!

**Pull request merged**